resources/sass/modules/_dashboard.scss

```diff
@@ -47,4 +47,49 @@
             }
         }
     }
+
+    &-modules {
+        &__module {
```

Comment:
i think that the property inside of `dashboard-modules__module` could just be in `dashboard-modules` i tried it and it does the same effect